src/main/java/com/linkedin/kmf/KafkaMonitor.java

```diff
@@ -66,6 +78,19 @@ public KafkaMonitor(Map<String, Map> testProps) throws Exception {
       }
     }
     _executor = Executors.newSingleThreadScheduledExecutor();
+
+    _offlineRunnables = new ConcurrentHashMap<>();
+    List<MetricsReporter> reporters = new ArrayList<>();
+    reporters.add(new JmxReporter(JMX_PREFIX));
```

Comment:
Yes, the existing services (e.g. `ProduceService`) will report JMX metrics using similar code. But `KafkaMonitor.java` doesn't report metrics as of now. Does this answer your question?